components/picker/stories/template.js

```diff
@@ -204,5 +206,15 @@ export const Template = ({
 			customStyles: customPopoverStyles,
 			content,
 		})}
+		${withSwitch ?
+			Switch({
+				...globals,
+				size,
+				label: "Toggle switch",
+				customStyles: {
+					"padding-inline-start": "15px"
+				}
+			})
+			: null}
```

Comment:
I wonder if we could use lit's `when` here instead of a ternary 🤔 Would look something like:

```
  ${when(withSwitch, () => 
			Switch({
				...globals,
				size,
				label: "Toggle switch",
				customStyles: {
					"padding-inline-start": "15px"
				}
			})
  }
```